x-pack/plugins/security_solution/server/lib/detection_engine/signals/utils.ts

```diff
@@ -83,6 +86,24 @@ export const shorthandMap = {
   },
 };
 
+export const getTraceIds = () => ({
```

Comment:
It looks like wouldn't need to set trace ids manually after https://github.com/elastic/kibana/pull/112973